debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -205,14 +159,6 @@ public class JdbcSinkConnectorConfig {
             .withDescription("A comma-separated list of primary key field names. " +
                     "This is interpreted differently depending on " + PRIMARY_KEY_MODE + ".");
 
-    public static final Field DATABASE_TIME_ZONE_FIELD = Field.create(DATABASE_TIME_ZONE)
```

Comment:
Please make sure that no config options are renamed without deprecation period, especially  when the change is going into micro release. I guess the same problem is with `TABLE_NAME_FORMAT_FIELD`.

Reply:
Sure, I had this on my list. Should we add an `alias` and/or `deprecated alias` for property names? With `deprecated alias` we could automate deprecation warnings. wdyt @jpechane

Reply:
@rk3rn3r Neat idea!

Reply:
Done